frame/staking-rewards/src/lib.rs

```diff
@@ -126,30 +130,41 @@ pub mod pallet {
 			amount: T::Balance,
 			/// Duration of stake.
 			duration_preset: DurationSeconds,
-			/// Position Id of newly created stake.
-			position_id: T::PositionId,
+			/// FNFT Collection Id
+			fnft_collection_id: T::AssetId,
+			/// FNFT Instance Id
+			fnft_instance_id: T::FinancialNftInstanceId,
 			keep_alive: bool,
 		},
 		Claimed {
 			/// Owner of the stake.
 			owner: T::AccountId,
-			/// Position Id.
-			position_id: T::PositionId,
+			/// FNFT Collection Id
+			fnft_collection_id: T::AssetId,
+			/// FNFT Instance Id
+			fnft_instance_id: T::FinancialNftInstanceId,
 		},
 		StakeAmountExtended {
-			position_id: T::PositionId,
+			/// FNFT Collection Id
+			fnft_collection_id: T::AssetId,
+			/// FNFT Instance Id
+			fnft_instance_id: T::FinancialNftInstanceId,
 			/// Extended amount
 			amount: T::Balance,
 		},
 		Unstaked {
 			/// Owner of the stake.
 			owner: T::AccountId,
-			/// Position Id of newly created stake.
-			position_id: T::PositionId,
+			/// FNFT Collection Id
+			fnft_collection_id: T::AssetId,
+			/// FNFT Instance Id
+			fnft_instance_id: T::FinancialNftInstanceId,
 		},
 		/// Split stake position into two positions
 		SplitPosition {
-			positions: Vec<(T::PositionId, BalanceOf<T>)>,
+			// REVIEW(benluelo,poisonphang): Should this be [_; 2] ? or perhaps two separate fields
```

Comment:
It's only the newly created position that we need to emit. This was reviewed with @andresmechali before

Reply:
I've updated the comment and we can change it later :+1: